include/frg/formatting.hpp

```diff
@@ -702,4 +727,15 @@ void format(const T &object, format_options fo, F &formatter) {
 
 } // namespace frg
 
+#ifdef FRIGG_HAS_IOSTREAM
+
+template<typename ...T>
+inline std::ostream &operator<<(std::ostream &out, frg::detail_::fmt_impl<T...> object) {
```

Comment:
this was intentionally left as a forwarding reference in my original suggestion

Reply:
oh, wait, it'll need a concept there to check for it's validity, because otherwise it will clash for everything